setup.py

```diff
@@ -40,13 +40,17 @@
         'Operating System :: OS Independent',
     ],
     python_requires='>=3.6',
-    install_requires=['numpy>=1.21',
-                      'rioxarray>=0.4',
-                      'pdal>=2.2',
-                      'geopandas>=0.9',
-                      'shapely>=1.7',
-                      'scipy>=1.6',
-                      'pathlib>=1.0'],
+    install_requires=['pyproj==3.0.1',
+                      'numpy==1.21.0',
+                      'rioxarray==0.4.2',
+                      'pdal==2.2.0',
+                      'python-pdal==2.4.0',
+                      'geopandas==0.9.0',
+                      'shapely==1.7.1',
+                      'scipy==1.6.3',
+                      'requests==2.25.1',
+                      'boto3==1.17.105',
+                      'matplotlib==3.4.2'],
```

Comment:
These versions can be relaxed a bit I think, e.g. was `'numpy>=1.21'`, now it could be `'numpy==1.21.*'` so that users automatically get the latest bug fixes...